readthedocs/projects/models.py

```diff
@@ -697,29 +691,20 @@ def all_active_versions(self):
         """
         return self.versions.filter(active=True)
 
-    def supported_versions(self, flat=True):
+    def supported_versions(self):
         """
         Get the list of supported versions.
         Returns a list of version strings.
         """
         if not self.num_major or not self.num_minor or not self.num_point:
```

Comment:
Good question indeed. Perhaps we can move that to another ticket about this issue, and get this out.